Xamarin.Forms.Core/Element.cs

```diff
@@ -107,10 +107,9 @@ public string StyleId
 			}
 		}
 
-		internal virtual ReadOnlyCollection<Element> LogicalChildren
-		{
-			get { return EmptyChildren; }
-		}
+		internal virtual ReadOnlyCollection<Element> LogicalChildrenInternal => EmptyChildren;
```

Comment:
You may have already considered this but interface definitions can be "overridden" like a virtual method by declaring the interface on the derived class and in that way we wouldn't need `Internal` methods. There are drawbacks such as having to implement all the other interface methods and not being as explicit without the `override` keyword. So up to you. https://gist.github.com/kingces95/47a4308e1597e149a214eb6e0daa9d5d